### PR TITLE
Condensing SHH profile(s) with sub-profiles

### PR DESCRIPTION
**Pull request by jfy133**
Following the (surprisingly simple) wizardary of https://github.com/nf-core/configs/pull/100

### CODE REVIEW

File: conf/shh.config [hunk 1]
```diff
@@ -22,9 +22,25 @@ executor {
...
   max_time = 720.h
   //Illumina iGenomes reference file path
   igenomes_base = "/projects1/public_data/igenomes/"
 }
+
+
+profiles {
+  cdag {
+      config_profile_description = 'MPI-SHH CDAG profile, provided by nf-core/configs.'
+      max_memory = 256.GB
```
Comment (MaxUlysse) [inline conf/shh.config]:
```suggestion
```

Comment (MaxUlysse) [reply conf/shh.config]: already there by default

File: conf/shh.config [hunk 2]
```diff
@@ -22,9 +22,25 @@ executor {
...
+      max_cpus = 32
```
Comment (MaxUlysse) [inline conf/shh.config]:
```suggestion
```

File: conf/shh.config [hunk 3]
```diff
@@ -22,9 +22,25 @@ executor {
...
+      max_time = 720.h
```
Comment (MaxUlysse) [inline conf/shh.config]:
```suggestion
```

File: conf/shh.config [hunk 4]
```diff
@@ -22,9 +22,25 @@ executor {
...
+  }
+  sdag {
+      config_profile_description = 'MPI-SHH SDAG profile, provided by nf-core/configs.'
+      max_memory = 2.TB
+      max_cpus = 128
+      max_time = 720.h
```
Comment (MaxUlysse) [inline conf/shh.config]:
```suggestion
```

**Comment by jfy133** [comment]
FYI: I would prefer to keep the `cdag` sub-profile for consistency (and we might have different specs relatively soon), even if it follows the 'default' SHH parameters.

**Pull request merged**